src/main/java/org/sagebionetworks/bridge/services/ParticipantService.java

```diff
@@ -634,14 +629,15 @@ private void updateAccountAndRoles(Study study, Account account, StudyParticipan
             String value = participant.getAttributes().get(attribute);
             account.getAttributes().put(attribute, value);
         }
-        if (callerIsAdmin(callerRoles)) {
-            updateRoles(callerRoles, participant, account);
+        RequestContext requestContext = getRequestContext();
+        if (requestContext.isAdministrator()) {
+            updateRoles(requestContext, participant, account);
```

Comment:
Should we add the superadmin equivalents to adminCanCreateAllRoles() and adminCanUpdateAllRoles() in unit tests?

Reply:
Added superadminCanCreateEverybody() and superadminCanUpdateEverybody().